client/rest-high-level/src/main/java/org/elasticsearch/client/watcher/DeactivateWatchRequest.java

```diff
@@ -28,25 +28,27 @@
     private final String watchId;
 
     public DeactivateWatchRequest(String watchId) {
+
+        if (watchId == null) {
+            ValidationException exception = new ValidationException();
+            exception.addValidationError("watch id is missing");
+            throw exception;
+        } else if (PutWatchRequest.isValidId(watchId) == false) {
+            ValidationException exception = new ValidationException();
+            exception.addValidationError("watch id contains whitespace");
+            throw exception;
+        }
+
         this.watchId = watchId;
     }
 
     public String getWatchId() {
         return watchId;
     }
 
+     // as per discussion https://github.com/elastic/elasticsearch/pull/34192/files#r221994527, keeping validate method as a no-op relic
```

Comment:
Feel free to just remove the comment here and the whole method instead of just returning empty since the default does this already.